### ISSUE TITLE
Right side issues due to voidgrubs' score HUD update ? 

### ISSUE THREAD

**Issue by MathieuNeveu** [Dan-Shields/League-OCR-HUD#8]
Hi ! 
I have the same kind of issues concerning the right team golds count and drake / nash timers so i commented this issue instead of creating a new one.
As i understood that's an ia model reading the hud to write into files. Looks like the HUD has been modified by Riot recently and in the right side the tower count seems to be very close to the gold count. It may cause that the actuel model is maybe 'outdated' ? 
I took some screenshots of these cases, note that these errors are intermittents but occurs more often when the gold count is > 9K. 

**Nash and Drake timers are not filled.** 
![HUD BUG 4](https://github.com/Dan-Shields/League-OCR-HUD/assets/23449532/45d5f3bc-4411-4ec8-89be-7ffc20cc7a3d)
**Gold count missing the tens** 
![BUG HUD 2](https://github.com/Dan-Shields/League-OCR-HUD/assets/23449532/a78fa520-e1a7-4303-a250-e451034b1ee1)
![BUG HUD 3](https://github.com/Dan-Shields/League-OCR-HUD/assets/23449532/ad8b162d-b47b-423b-9be6-706deefa3be5)
![BUG HUD 1](https://github.com/Dan-Shields/League-OCR-HUD/assets/23449532/19bd1057-b26d-4917-a63e-0e91fba1c227)

Have a good day !

_Originally posted by @MathieuNeveu in https://github.com/Dan-Shields/League-OCR-HUD/issues/7#issuecomment-1969289696_

**Comment by Dan-Shields** [comment]
This can be fixed by updating the [sectors.json](https://github.com/Dan-Shields/League-OCR-HUD/blob/960e3ae3f32c48389c426509aa7252acf44daadd/sectors.json) file using the Sectors.psd photoshop file as a guide.

If you have time @MathieuNeveu to take a fresh screenshot of the game, update that in the photoshop file and then adjust the sectors definitions that'd be great if you could make a pull request. Shouldn't take more than 30 minutes.

**Comment by MathieuNeveu** [comment]
🎉
Working on it ! I will make a PR in the following days, successfully tried to add the voidgrubs sectors on the json and mask so we can have the outputs left_voidgrubs.txt & right_voidgrubs.txt 👍
About the drake & baron, seems that now timers are displayed on the bottom right of the screen so we don't have an easy fix for it ?

**Comment by GeNeRaLcArRoT** [comment]
@MathieuNeveu Hello, I'm having the same problems, have you gotten a solution going? I'd very much appreciate the help. Also regarding the baron and drake timers, there is an in game setting for your spectator mode to move the timers into the top corners, if that's what you were having issues with.

**Comment by MathieuNeveu** [comment]
@GeNeRaLcArRoT Yes, i will push something today without dealing with timers. I didn't find out the option that your talking about timers, if you could send me a screenshot i'll be happy :)

**Comment by MathieuNeveu** [comment]
uops looks like i don't have the rights to push on the repos @Dan-Shields 😬
i named my branch fix/voidgrubs-repositioning don't know if there is some rules to respect on the repos

**Comment by GeNeRaLcArRoT** [comment]
@MathieuNeveu To turn the neutral timers into the dragon and baron timers used in pro play, open the game.cfg file in the same config folder. Simply add "[Spectator] eSportsNeutralTimers=1" and when you enable the timers in spectate will appear as they do on the professional streams. https://leagueoflegends.fandom.com/wiki/Spectator_Mode (just above the comments in this wiki post)

I also saw on a reddit post that writing it like this 
[Spectator] 
eSportsNeutralTimers=1

Should work. If it doesn't @ me here again.

**Comment by MathieuNeveu** [comment]
Hi
Since i couldn't push the fix on this repo due to lack of rights i v pushed it on this fork @GeNeRaLcArRoT :  https://github.com/MathieuNeveu/League-OCR-HUD/tree/fix/voidgrubs-repositioning

Still without the drakes timers, will take a look today or tomorrow 

Have a good day 🌞

**Comment by Dan-Shields** [comment]
@MathieuNeveu you shouldn't have push rights to this repo. You've done the right thing and made a fork, now if you create a pull request to this repository I can review your code.

@GeNeRaLcArRoT is right about the neutral timers!

@Fill30 can you review Mathieu's PR when he's made it?
